### PR TITLE
Added testcase to validate 3495 timer for both default and dedicated bearer while mme restarts

### PR DESCRIPTION
**Pull request by rsarwad**
[AGW]: Added testcase to validate 3495 timer for both default and dedicated bearer while mme restarts
## Summary
Added testcase to validate 3495 timer for both default and dedicated bearer while mme restarts

## Test Plan
Executed s1ap sanity test suite

Dependent PR is, https://github.com/magma/magma/pull/4017

### CODE REVIEW

File: lte/gateway/python/integ_tests/s1aptests/test_3485_timer_for_dedicated_bearer_with_mme_restart.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        Step1: UE attaches to network
+        Step2: Send an indication to S1ap stack to drop E-Rab Setup
+               Request message, sent as part of secondary PDN activation
+               procedure.
+        Step3: Send an indication to initiate Dedicated bearer activation
+        Step4: Send an indication to S1ap stack to not to drop E-Rab Setup
+               Request message, so that re-transmitted message reaches to
+               TFW
+        Step5: Send command to Magma to restart mme service
+        Step6: TFW shall receive re-transmitted Actiavte Dedicated EPS
```
Comment (ulaskozat) [inline lte/gateway/python/integ_tests/s1aptests/test_3485_timer_for_dedicated_bearer_with_mme_restart.py]: Typo Actiavte->Activate

File: lte/gateway/python/integ_tests/s1aptests/test_3485_timer_for_dedicated_bearer_with_mme_restart.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+               Bearer Context Request and send Actiavte Dedicated EPS Bearer
```
Comment (ulaskozat) [inline lte/gateway/python/integ_tests/s1aptests/test_3485_timer_for_dedicated_bearer_with_mme_restart.py]: Same typo as above.

File: lte/gateway/python/integ_tests/s1aptests/test_3485_timer_for_dedicated_bearer_with_mme_restart.py [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+               Context Accept
+        Step7: TFW shall initaite de-activation of deidicated bearer and then
```
Comment (ulaskozat) [inline lte/gateway/python/integ_tests/s1aptests/test_3485_timer_for_dedicated_bearer_with_mme_restart.py]: typos initaite->initiate and deidicated -> dedicated

File: lte/gateway/python/integ_tests/s1aptests/test_3485_timer_for_default_bearer_with_mme_restart.py [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+        self._s1ap_wrapper.cleanup()
+
+    def test_3485_timer_for_default_bearer_with_mme_restart(self):
+        """ Test case validates the functionality of 3485 timer for
+            default bearer while MME restarts
+        Step1: UE attaches to network
+        Step2: Send an indication to S1ap stack to drop E-Rab Setup
+               Request message, sent as part of secondary PDN activation
+               procedure.
+        Step3: Initaite activation of secondary PDN
```
Comment (ulaskozat) [inline lte/gateway/python/integ_tests/s1aptests/test_3485_timer_for_default_bearer_with_mme_restart.py]: typo Initaite -> Initiate

File: lte/gateway/python/integ_tests/s1aptests/test_3485_timer_for_default_bearer_with_mme_restart.py [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+        self.assertEqual(
+            response.msg_type, s1ap_types.tfwCmd.UE_ERAB_DROP_IND.value
+        )
+        print(
+            "*******************Received ERAB_DROP_IND for last received ",
+            "E-Rab Setup Request for secondary PDN's default bearer ",
+        )
+
+        print(
+            "*** Sending indication to disbale dropping of Erab Setup Req",
```
Comment (ulaskozat) [inline lte/gateway/python/integ_tests/s1aptests/test_3485_timer_for_default_bearer_with_mme_restart.py]: typo disbale -> disable

**Review by ulaskozat — CHANGES_REQUESTED** [review]
Fix the typos.

**Comment by ulaskozat** [comment]
@rsarwad could you take care of codacy warnings?

**Review by ulaskozat — APPROVED** [review]
LGTM with nits to be completed from codacy warnings.

**Comment by ssanadhya** [comment]
@rsarwad , could you please remove the T3485 test cases from this PR as they need changes on MME? The T3495 tests are good to merge once the T3485 tests are removed.

**Review by ssanadhya — APPROVED** [review]

**Pull request merged**